Svg/Basic Shapes/SvgImage.cs

```diff
@@ -260,7 +261,7 @@ protected override void Render(ISvgRenderer renderer)
                         renderer.PopBoundable();
                     }
 
-                    if (this.ClipPath != null || !string.IsNullOrEmpty(this.Clip))
+                    if(Parent is not SvgUse)
```

Comment:
aha! so with a svguse, we must keep the clip?

Reply:
Usually svg use has the image as an attribute and not child. Thats  why it should not clip again.